website/src/components/props-table.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react"
+// @ts-ignore
```

Comment:
Looks like this may not be needed anymore?

Reply:
You are right, I removed it ✅